libs/ledgerjs/packages/hw-app-eth/src/Eth.ts

```diff
@@ -1205,9 +1205,6 @@ export default class Eth {
   }
 
   provideERC20TokenInformation({ data }: { data: Buffer }): Promise<boolean> {
-    console.warn(
-      "hw-app-eth: eth.provideERC20TokenInformation is deprecated. signTransaction solves this for you when providing it in `resolution`."
-    );
```

Comment:
Should be replaced with just a comment maybe? So info isn't lost

Reply:
what do you mean? To me this was accidentally made deprecated, it's actually legit API in the contract, e.g. for stark usage?

Reply:
Ah ok, wondering why it was logged as deprecated, but if that's not true then it should be removed indeed.

Reply:
when i introduced the resolve mechanism, i have wrote this deprecating assuming that all external usage of this method would no longer be needed but i didn't see the stark usage back at the time